test/models/search_test.dart

```diff
@@ -0,0 +1,72 @@
+library search_test;
+
+import 'package:nhentai/data_model.dart';
+import 'package:test/expect.dart';
+import 'package:test/scaffolding.dart';
+
+import 'book_test.dart' as book_test;
+
+const bookTitleJson = '{"english":"[Puu no Puupuupuu (Puuzaki Puuna)] Hitozukiai ga Nigate na Miboujin no Yukionna-san to Noroi no Yubiwa [English]","japanese":"[ぷぅのぷぅぷぅぷぅ (ぷぅ崎ぷぅ奈)] 人付き合いが苦手な未亡人の雪女さんと呪いの指輪 [英訳]","pretty":"Hitozukiai ga Nigate na Miboujin no Yukionna-san to Noroi no Yubiwa"}';
+
+const titleEnglish = '[Puu no Puupuupuu (Puuzaki Puuna)] Hitozukiai ga Nigate na Miboujin no Yukionna-san to Noroi no Yubiwa [English]';
+const titleJapanese = '[ぷぅのぷぅぷぅぷぅ (ぷぅ崎ぷぅ奈)] 人付き合いが苦手な未亡人の雪女さんと呪いの指輪 [英訳]';
+const titlePretty = 'Hitozukiai ga Nigate na Miboujin no Yukionna-san to Noroi no Yubiwa';
```

Comment:
use some artificial data, we don't need to be affiliated with anything particular.